Makefile

```diff
@@ -16,6 +16,10 @@ test:
 go-build:
 	go build -o $(APP_NAME) main.go
 
+build-SSOSyncFunction:
+	GOOS=linux GOARCH=arm64 go build -o bootstrap main.go
+	cp ./bootstrap $(ARTIFACTS_DIR)/.
```

Comment:
NIT: Would it be possible to document the complete build&deploy process for this code, which includes this new function? 👀